aztk/node_scripts/install/install.py

```diff
@@ -41,6 +41,7 @@ def setup_host(docker_repo: str):
 
     cluster_conf = read_cluster_config()
 
+    #TODO share azure file mounts
```

Comment:
?

Reply:
I mean we should eventually share just the file shares that are specified, instead of all /mnt